OMCompiler/SimulationRuntime/c/simulation/solver/gbode_nls.h

```diff
@@ -43,8 +43,6 @@
 extern "C" {
 #endif
 
-void initializeStaticNLSData_SR(DATA* data, threadData_t *threadData, NONLINEAR_SYSTEM_DATA* nonlinsys, modelica_boolean initSparsePattern);
```

Comment:
If you remove this function prototype add a "private" one at the beginning of `gbode_nls.c`.